src/pvnet_app/forecast_compiler.py

```diff
@@ -40,13 +42,15 @@
 
 
 def validate_forecast(
+    forecast_data: Dict[str, np.ndarray],
     national_forecast_values: np.ndarray,
     national_capacity: float,
     logger_func: Callable[[str], None],
 ) -> None:
     """Checks various conditions using the full forecast values (in MW).
 
     Args:
+        forecast_data: A dictionary containing 'datetime', 'forecast_values', 'lat', 'lon'.
```

Comment:
I would prefer if national_forecast_values, could be changed to a pd.Series with datetimes. Then there is less to pass in. 

For lat lon, we can just take the centre of the UK for the moment. Something like 55.3781° N, 3.4360° W

Reply:
you'll need to update where this function is called too